c8run/go.mod

```
@@ -0,0 +1,3 @@
+module c8run
```

```suggestion
module github.com/camunda/camunda/c8run
```

This would be the typical naming for a go module. 
`module github.com/org-name/repo-name/path/to/module-dir`

Then you can also get `go install` to work - (not sure if it also would work with the other name).  

So you can do the following:
```
go install github.com/camunda/camunda/c8run (installs c8run to your go path)
c8run (run camunda!!! 🏃 )
```

---

I think I did this in https://github.com/camunda/camunda/pull/24040/commits/1b49a0ea11314c6e68f18770c3f98146a8f420dd but it's hard to tell because I don't think I can reference a branch name in a `go install` command.

I'm also unsure how versioning works in golang. is it all just git tags? if so then well, I don't control the git tags applied in the camunda monorepo. it's worth a try though.